draft-ietf-jsonpath-base.md

```diff
@@ -2178,8 +2196,8 @@ JSONPath:
 * Square brackets in XPath expressions always operate on the *node
   set* resulting from the previous path fragment. Indices always start
   at 1.
-* With JSONPath, square brackets operate on the *object* or *array*
-  addressed by the previous path fragment. Array indices always start
+* With JSONPath, square brackets operate on the *nodelist*
```

Comment:
This (in particular with the array index note after it) is a bit misleading -- can we say that it operates on each of the nodes in the nodelist?

Reply:
Yes. Adopted.